lib/modules/apostrophe-pages/lib/api.js

```diff
@@ -1168,6 +1168,12 @@ module.exports = function(self, options) {
           return item.action !== 'insert-page';
         });
       }
+
+      if (!args.edit && !args.contextMenu && self.apos.permissions.can(req, 'view-page')) {
+        args.contextMenu = _.filter(self.options.contextMenu, function(item) {
+          return item.action === 'copy-page'
```

Comment:
Is this about allowing the user to start from the page they want to copy?

The idea is that you still start from the page you want to be the new parent, but there's a join inside the page settings modal which defaults to that page but can be used to pick a different source to copy from.